docs/introduction/api.md

```diff
@@ -0,0 +1,67 @@
+# Open Food Facts API Documentation
+
+Everything you need to know about Open Food Facts API.
+
+## Overview
+
+Open Food Facts is a food products database made by everyone, for everyone, that can help you make better food choices. Seeing it is open data, anyone can reuse it for any purpose. For example, you are building a nutrition app.
+The Open Food Facts API enables developers to add to the products database and retrieve information about existing products. You may use the API to build applications allowing users to contribute to the database and make healthier food choices.
+The current version of the API is `2`.
+
+<!--- We can add a disclaimer image beside the next paragraph instead of making it a subheading -->
+
+Data in the Open Food Facts database is provided voluntarily by users who want to support the program. As a result, there are no assurances that the data is accurate, complete, or reliable. The user assumes the entire risk of using the data.
+
```

Comment:
I think we shall add a **Before you start** section, that use same text as on https://world.openfoodfacts.org/data.

A first section for license (as on data page)

A second section on "how to best use the API" taking text on /data under *Wrappers and SDKs*